10379.xml

```diff
@@ -0,0 +1,129 @@
+FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+POSSIBILITY OF SUCH DAMAGE.
+-->
+
+<LWM2M xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://openmobilealliance.org/tech/profiles/LWM2M.xsd">
+    <Object ObjectType="MODefinition">
+        <Name>Measurement Metadata</Name>
+        <Description1>This object contains measurement metadata for sensor objects. 
+The corresponding sensor is identified with an object link.
+Measurement Quality Indicator and Measurement Quality Level values provided in this object have a precedence over 
+values provided in a sensor object. If multiple object instances link to the same sensor object instance then 
+the metadata provided by the largest instance id takes precedence.
+        </Description1>
+        <ObjectID>10379</ObjectID>
+        <ObjectURN>urn:oma:lwm2m:x:10379</ObjectURN>
+        <LWM2MVersion>1.1</LWM2MVersion>
+        <ObjectVersion>1.0</ObjectVersion>
+        <MultipleInstances>Multiple</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Resources>
+            <Item ID="0">
+                <Name>Linked Sensor</Name>
+                <Operations>R</Operations>
+                <MultipleInstances>Single</MultipleInstances>
+                <Mandatory>Mandatory</Mandatory>
+                <Type>Objlnk</Type>
+                <RangeEnumeration></RangeEnumeration>
+                <Units></Units>
+                <Description>Link to the sensor object instance that this metadata relates to.
+                </Description>
+            </Item>
+            <Item ID="1">
+                <Name>Measurement Quality Reason Code</Name>
+                <Operations>R</Operations>
+                <MultipleInstances>Multiple</MultipleInstances>
+                <Mandatory>Optional</Mandatory>
+                <Type>Integer</Type>
+                <RangeEnumeration>0..1024</RangeEnumeration>
+                <Units></Units>
+                <Description>Measurement quality degradation reason reported by a smart sensor. 
+A degraded measurement has Measurement Quality Indicator value 1-3 or Measurement Quality Level less than 100.
+0: UNKNOWN Reason is unknown.
+1: UNDER_RANGE Measured value is under measurement range. 
+2: OVER_RANGE Measured value is over measurement range.
+3: NOISY Measured value seems to change too quickly. 
+4: STUCK_VALUE Measured value seems to change too slowly.
+5: STEP_TOO_HIGH Measured value has changed too much.
```

Comment:
Changed too much from the previous measurement? Given it's "step". Just to make clear the difference to "too quickly".